src/main/scala/io/opentargets/etl/backend/cancerbiomarkers/CancerBiomarkers.scala

```diff
@@ -0,0 +1,168 @@
+package io.opentargets.etl.backend.cancerbiomarkers
+
+import io.opentargets.etl.backend.spark.IoHelpers.IOResources
+import io.opentargets.etl.backend.spark.{IOResource, IOResourceConfig, IoHelpers}
+import io.opentargets.etl.backend.{Configuration, ETLSessionContext}
+import org.apache.spark.sql.functions._
+import org.apache.spark.sql.types.LongType
+import org.apache.spark.sql.{DataFrame, Dataset, SparkSession}
+
+case class CancerSource(name: String, description: String, link: String)
```

Comment:
@JarrodBaker did you check in the data that there are no empty strings in there? having a description field that is always filled make me wonder whether is something that happens in this reality

Reply:
It's a manually curated file (your favourite) which was previously hidden in the Python source code: all fields exist.